stdlib/public/Concurrency/Actor.swift

```diff
@@ -72,3 +72,9 @@ public func _defaultActorDestroy(_ actor: AnyObject)
 @_silgen_name("swift_task_enqueueMainExecutor")
 @usableFromInline
 internal func _enqueueOnMain(_ job: UnownedJob)
+
+/// Produce a reference to the actor to which the enclosing code is
+/// isolated, or `nil` if the code is nonisolated.
+@available(SwiftStdlib 5.1, *)
+@freestanding(expression)
+public macro isolation() -> (any AnyActor)? = Builtin.IsolationMacro
```

Comment:
This [just broke the Android community CI](https://ci-external.swift.org/job/oss-swift-RA-linux-ubuntu-16.04-android-arm64/6303/console), which doesn't have a prebuilt host Swift compiler so the trunk compiler it builds doesn't use the swift-syntax parser and doesn't support macros:
```
swift/stdlib/public/Concurrency/Actor.swift:80:14: error: macros are not supported in this compiler
public macro isolation() -> (any AnyActor)? = Builtin.IsolationMacro
             ^
```
Other uses of macros in the stdlib [have put in checks to avoid this problem](https://github.com/apple/swift/pull/70351/files#diff-3f0f9a0e4da3abbc9c10ee7fcbd091f6ba64491d6998e81a2edb55e24e75249bR16), maybe something similar would work here?